CRM/Report/Form.php

```diff
@@ -3576,11 +3599,14 @@ public function postProcess() {
   /**
    * Set limit.
    *
-   * @param int $rowCount
+   * @param int|null $rowCount
    *
    * @return array
    */
-  public function limit($rowCount = self::ROW_COUNT_LIMIT) {
+  public function limit($rowCount = NULL) {
+    if (CRM_Utils_System::isNull($rowCount)) {
```

Comment:
I see isNull isn't deprecated yet but it does unexpected things sometimes. Was this copy/paste or is there a reason to use it?

Reply:
I was wanting to get the function if $rowCount was NULL rather than say 0 had been passed into the function

Reply:
In php 7.4 there's a nifty `$rowCount ??= $this->getRowCount();`. For php 7 in general could do `$rowCount = $rowcount ?? $this->getRowCount();` or there's regular `is_null()`.